INN/umbrella-borderzine
# Create new borderzine_byline class and relevant templates + styles

## Changes

This pull request makes the following changes:

- Creates new `Borderzine_Byline` class which is copied from `Largo_Byline`, and reworks the `generate_byline` method to reorder byline sections + includes new `excerpt()` method that is borrowed from the `largo-author-bio.php` widget.
- Adds some post page specific styling modifications for the byline

Desktop:
![Screen Shot 2019-09-17 at 12 38 01 PM](https://user-images.githubusercontent.com/18353636/65062036-2bab5700-d949-11e9-8a48-1a26047903ef.png)

Mobile:
![Screen Shot 2019-09-17 at 12 46 22 PM](https://user-images.githubusercontent.com/18353636/65062035-2bab5700-d949-11e9-9222-f76ebc80640f.png)

## Why



For #3

## Testing/Questions

Features that this PR affects:

- Bylines


Questions that need to be answered before merging:

- [ ] Does it work ok?
- [ ] @benlk Any weird edge cases we should be worried about here?

Steps to test this PR:

1. View a post with an author without an avatar
2. View a post with an author with an avatar
